src/components/interaction-designer/block-editors/choices/SelectOneResponseBlockContactPropertyEditor.vue

```diff
@@ -85,9 +106,32 @@ export const SelectOneResponseBlockContactPropertyEditor = {
       return this.contactProperty?.data_type === 'number'
     },
 
+    isBooleanProperty() {
+      return this.contactProperty?.data_type === 'boolean'
+    },
+
     choices() {
       return this.block.config.choices
     },
+
+    isMultipleChoiceProperty() {
+      return this.contactProperty?.data_type === 'multiple_choice'
+    },
+
+    choiceValueOptions() {
+      return [
+        {value: '', description: this.trans('flow-builder.no-value')},
+        ...(this.contactProperty?.choices ?? []),
+      ]
+    },
+
+    booleanChoiceValueOptions() {
+      return [
+        {value: '', description: this.trans('flow-builder.no-value')},
+        {value: 'TRUE', description: this.trans('flow-builder.true')},
+        {value: 'FALSE', description: this.trans('flow-builder.false')},
+      ]
+    },
```

Comment:
I wonder if we should fail validation when 'No value' is selected (for both boolean and multiple-choice).
Did you see anything about it in the AC/flowcharts? (If not - would you like to ask Nancy?)

Reply:
Asked the question in [VMO-6785](https://viamoinc.atlassian.net/browse/VMO-6785?focusedCommentId=21058)